### PR TITLE
When creating a new user, require a password match

### PR DESCRIPTION
**Pull request by ao3742**
Improvement to femr-159 has been completed.

### CODE REVIEW

File: app/femr/ui/views/admin/users/create.scala.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
             'class -> "fInput",
             'placeholder -> "Email Address",
             '_label -> "Email Address",
-            '_isRequired -> true
+            '_isequired -> true
```
Comment (kdunlap) [inline app/femr/ui/views/admin/users/create.scala.html]: I think this line was inadvertently changed. It should read:

```
'_isRequired -> true
```

**Comment by kdunlap** [comment]
Outside of the minor typo, this looks good and works well for me.

**Comment by kevinzurek** [comment]
I think this was re-opened here: https://github.com/FEMR/femr/pull/524...

**Comment by ao3742** [comment]
Hi Kevin,

I thought I deployed correct version in Improvement femr159 new user require password match ver2<https://github.com/FEMR/femr/pull/524> #524

I will submit it again now.


Thanks,
Vivek

**Pull request closed without merging**